pages/recordbag.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import RecordFile from '../components/RecordFile';
+
+export default function RecordBag({ selectionState, onHandleChange }) {
+  const selection = selectionState;
```

Comment:
I would rather rename the prop while destructuring it:
```suggestion
export default function RecordBag({ selectionState: selection, onHandleChange }) {
```

Even better: change the prop´s name in your `_app.js` from `selectionState` to `selection`.